pyanaconda/core/constants.py

```diff
@@ -486,3 +486,7 @@ class PayloadRequirementType(Enum):
 
 # Path to the System Purpose configuration file on a system.
 RHSM_SYSPURPOSE_FILE_PATH = "/etc/rhsm/syspurpose/syspurpose.json"
+
+# GID and UID modes
+ID_MODE_USE_VALUE = "ID_MODE_USE_VALUE"
+ID_MODE_USE_DEFAULT = "ID_MODE_USE_DEFAULT"
```

Comment:
To reduce the confusion and possible gotchas about this "fake" enum, how about using a boolean instead? Or even more pythonic, set `self._gid = None` initially if it's not set? Then you don't need an explicit _mode in the first place.

Reply:
The `GroupData` and `UserData` classes are representations of DBus dictionaries, so we are limited by DBus data types. That is why we cannot use enums or set the properties to `None`.

Reply:
Btw. we have considered the boolean, but the majority voted for the string.

Reply:
Ack, thanks. Then let's fix the empty string modes.